custom_components/opnsense/sensor.py

```
@@ -41,322 +41,363 @@
+
+    for filesystem in dict_get(state, "telemetry.filesystems", []):
+        device_clean: str = normalize_filesystem_device_name(filesystem["device"])
+        mountpoint_clean: str = normalize_filesystem_device_name(
+            filesystem["mountpoint"]
+        )
+        entity = OPNsenseFilesystemSensor(
+            config_entry=config_entry,
+            coordinator=coordinator,
+            entity_description=SensorEntityDescription(
+                key=f"telemetry.filesystems.{device_clean}",
+                name="Filesystem Used Percentage {}".format(mountpoint_clean),
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)